master/reference/advanced/etcd-rbac/kubernetes.md

```diff
@@ -1,53 +1,53 @@
 ---
-title: Using etcd RBAC to segment Kubernetes and Calico
+title: Using Etcd RBAC to segment Kubernetes and Calico
 ---
 
-When using etcd with RBAC, all components that access etcd must be configured
-with the proper certificates.  This document describes the users and roles
-needed to segment etcd so that Kubernetes and Calico can only read and write
-within their respected subtrees/prefixes.  To configure more compartmentalized
-configurations of the Calico components, see this addon
+When using Etcd with RBAC, all components that access Etcd must be configured
+with the proper certificates. This document describes the users and roles
+needed to segment Etcd so that Kubernetes and Calico can only read and write
+within their respected subtrees/prefixes. To configure more compartmentalized
+configurations of the Calico components, see this addon:
 [guide](kubernetes-advanced).
 
 This guide assumes you are following the general
-[Setting up etcd certificates for RBAC guide](index) and using its guidance
-for setting up certificates and etcd cluster, users, and roles.
+[Setting up Etcd certificates for RBAC guide](index) and using its guidance
+for setting up certificates and Etcd cluster, users, and roles.
 
 ## Why you might be interested in this guide
 
-You are using Kubernetes and Calico who share an etcd datastore and wish
-to ensure that Calico and Kubernetes are unable to access each other's etcd
+You are using Kubernetes and Calico that share an Etcd datastore and you wish
+to ensure that Calico and Kubernetes are unable to access each others' Etcd
 data.
 
-## Needed etcd Roles
+## Needed Etcd Roles
 
-The following components need certificates with a Common Name that matches an
-etcd user that has been given appropriate roles allowing access to the key
+The following components need certificates with a Common Name that match an
```

Comment:
I think "matches" is more correct actually....because this verb refers to "common name", which is singular.

Reply:
Wasn't sure about this one. I read it as "The following components (blah blah) match ..." but that's wrong in hindsight.